### DISCUSSION TITLE
https access when create an embeded to a html page?

### DISCUSSION THREAD

**Discussion by claclin** [FlowiseAI/Flowise#1592] [Q&A] [answered]
Hi team, I have create a chat and I want to embed it on my webpage. My WebPage has the access https// but when I embed the code on my page, this code is http://localhost:portnumber, if I modify the localhost for the url of my WebPage and writes https instead of http it doesn´t work. How can I create a https url for my chat?? Thanks in advance.

**Comment by Jaredude** [comment]
Use [ngrok](https://ngrok.com)

**Reply by claclin** [reply]
more specific please?

**Reply by Jaredude** [reply, marked as answer]
Your use case is exactly what ngrok is made for: allowing tunneling to a local machine from outside networks. [Install ngrok](https://ngrok.com/docs/getting-started/) and run it using the port Flowise is running on your local machine, and it will provide you with an https tunnel to connect to your local machine.

If, instead, you are looking to go into Production with your bot, you will need to [install Flowise on a production server](https://docs.flowiseai.com/configuration/deployment) with permanent storage or a DB connection. Then configure that server with a secure cert. The [AWS documentation](https://docs.flowiseai.com/configuration/deployment/aws#install-certbot-to-have-https) is fairly good, and you should be able to get it setup in less than 30 minutes.

Hope that helps.

**Comment by claclin** [comment]
Thanks I will try ngrok. When you say install Flowise on a production
server you mean in AWS or AZURE, so on a cloud server?

El vie, 26 ene 2024 a las 18:14, Jared Tracy ***@***.***>)
escribió:

> Your use case is exactly what ngrok is made for: allowing tunneling to a
> local machine from outside networks. Install ngrok
> <https://ngrok.com/docs/getting-started/> and run it using the port
> Flowise is running on your local machine, and it will provide you with an
> https tunnel to connect to your local machine.
>
> If, instead, you are looking to go into Production with your bot, you will
> need to install Flowise on a production server
> <https://docs.flowiseai.com/configuration/deployment> with permanent
> storage or a DB connection. Then configure that server with a secure cert.
> The AWS documentation
> <https://docs.flowiseai.com/configuration/deployment/aws#install-certbot-to-have-https>
> is fairly good, and you should be able to get it setup in less than 30
> minutes.
>
> Hope that helps.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FlowiseAI/Flowise/discussions/1592#discussioncomment-8259815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2LHOFCLDGAMCDCP42TOD2TYQPP7PAVCNFSM6AAAAABCGXQLB2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DENJZHAYTK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Saludos,

Claudio

**Reply by Jaredude** [reply]
Yes, for running in production, you wouldn't have Flowise running on your local machine using ngrok. You'd create a cloud server and install AWS on it using the steps defined in the documents linked previously.

**Comment by claclin** [comment]
Thanks!!!

El sáb, 27 ene 2024 a las 19:15, Jared Tracy ***@***.***>)
escribió:

> Yes, for running in production, you wouldn't have Flowise running on your
> local machine using ngrok. You'd create a cloud server and install AWS on
> it using the steps defined in the documents linked previously.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FlowiseAI/Flowise/discussions/1592#discussioncomment-8266731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2LHOFDFPUJJMXR3BUUQVRDYQU75HAVCNFSM6AAAAABCGXQLB2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DENRWG4ZTC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Saludos,

Claudio
